wrappers/python/examples/depth_ucal_example.py

```diff
@@ -0,0 +1,176 @@
+## License: Apache 2.0. See LICENSE file in root directory.
+## Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#####################################################
+##                   auto calibration              ##
+#####################################################
+
```

Comment:
Please consider to add some comment on what does this example does, by adding one explanation comment at the beginning or by adding comments along the code.

Reply:
Done